src/channels/03-music.yml

```
@@ -268,6 +268,10 @@ schedule:
     - "17:30":
     - "18:00":
     - "18:30":
+        title: drone-it
+        author:
+          - Patricio Cifredo
```

```suggestion
        author: Patricio Cifredo
```
Since there's just one author we can leave it all on the same line